### ISSUE TITLE
settings not saved

### ISSUE THREAD

**Issue by energiesgraphiques** [RxLaboratory/DuBlast-Maya#2]
whatever option I change, Dublast comes back to default settings when I open it again. (launch with mel command saved in a shelf)

**Comment by Nico-Duduf** [comment]
yes, that's something I still have to implement; I will very soon in a new version!
